public/sitemap.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is autogenerated, please delete.

Reply:
Done